website/docs/d/logs_router_tenant.html.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_logs_router_tenant"
+description: |-
+  Get information about logs_router_tenant
+subcategory: "IBM Logs Router"
```

Comment:
subcategory should be `Logs Router`. Kindly update in resource docs as well.

Reply:
Changed Subcategory to "Logs Router"